src/bcd/bcd/MultiscaleDenoiser.cpp

```diff
@@ -29,6 +29,62 @@ using namespace std;
 
 namespace bcd
 {
+namespace
+{
+
+class MultiScaleProgressReporter
```

Comment:
Indent content of anonymous namespace.

Our rule is pretty simple:

We only make an exception for top-level namespaces that span entire files. Indenting those would amount to indenting a whole file for no clear benefit. We always indent everything else.